### PR TITLE
feature-erms-2522

### PR DESCRIPTION
**Pull request by croqueGrec09**
various improvements on license view, see ERMS-2522

### CODE REVIEW

File: app/grails-app/views/license/members.gsp [hunk 1]
```diff
@@ -29,28 +29,71 @@
...
+            </div>
+            <div class="field fieldcontain">
+                <semui:datepicker label="default.valid_on.label" id="validOn" name="validOn" placeholder="filter.placeholder" value="${validOn}" />
+            </div>
+
+            <%
+                def fakeList = []
+                fakeList.addAll(RefdataCategory.getAllRefdataValues(RDConstants.SUBSCRIPTION_STATUS))
+                //fakeList.add(RefdataValue.getByValueAndCategory('subscription.status.no.status.set.but.null', 'filter.fake.values'))
+                fakeList.remove(RefdataValue.getByValueAndCategory('Deleted', RDConstants.SUBSCRIPTION_STATUS))
```
Comment (klodav) [inline app/grails-app/views/license/members.gsp:50]: There should be no subscription status "deleted"?

Comment (croqueGrec09) [reply app/grails-app/views/license/members.gsp]: Removed as with seven other occurrences

File: app/grails-app/controllers/com/k_int/kbplus/LicenseController.groovy [hunk 2]
```diff
@@ -116,7 +116,7 @@ class LicenseController extends AbstractDebugController {
             result << preCon
 
             // restrict visible for templates/links/orgLinksAsList
-            result.visibleOrgLinks = OrgRole.executeQuery('select oo from OrgRole oo where oo.lic = :license and oo.org != :context and oo.roleType not in (:roleTypes)',[license:result.license,context:result.institution,roleTypes:[OR_LICENSEE, OR_LICENSING_CONSORTIUM]])
+            result.visibleOrgLinks = OrgRole.executeQuery('select oo from OrgRole oo where oo.lic = :license and oo.org != :context and oo.roleType not in (:roleTypes) order by oo.roleType.value asc, oo.org.sortname asc',[license:result.license,context:result.institution,roleTypes:[OR_LICENSEE, OR_LICENSEE_CONS, OR_LICENSING_CONSORTIUM]])
```
Comment (klodav) [inline app/grails-app/controllers/com/k_int/kbplus/LicenseController.groovy:119]: Does not work with translations .. order by oo.roleType.**value_<en|de>** needed

Comment (croqueGrec09) [reply app/grails-app/controllers/com/k_int/kbplus/LicenseController.groovy]: Corrected with 5a0385a

File: app/grails-app/views/license/_actions.gsp [hunk 3]
```diff
@@ -2,63 +2,64 @@
...
+                    />
+                </g:if>
             </g:if>
 
-            <g:if test="${editable}">
-                <div class="divider"></div>
-                <g:link class="item" action="delete" id="${params.id}"><i class="trash alternate icon"></i> ${message(code:'deletion.license')}</g:link>
+            <div class="divider"></div>
+            <%
+                License license = License.get(license?.id)
```
Comment (klodav) [inline app/grails-app/views/license/_actions.gsp:31]: :roll_eyes: .. good, better, the best

Comment (croqueGrec09) [reply app/grails-app/views/license/_actions.gsp]: fixed with 85200c9

**Pull request merged**